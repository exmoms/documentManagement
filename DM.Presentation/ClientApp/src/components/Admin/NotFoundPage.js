import React from "react";

export default function () {
  return <div>404 Not found Page!</div>;
}
