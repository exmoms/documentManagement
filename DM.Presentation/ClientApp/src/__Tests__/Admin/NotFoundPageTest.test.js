
import React from "react";
import ReactDom from "react-dom";
import NotFoundPage from "../../components/Admin/NotFoundPage";
import { cleanup } from "@testing-library/react";
import { shallow , mount} from "enzyme";
import toJSON from "enzyme-to-json";

afterEach(cleanup);
describe("NotFoundPage", () => {
   afterEach(() => {
    jest.clearAllMocks();
  });
  test("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDom.render(<NotFoundPage/>,div
    );
  });

  test("matches snapshot", () => {
    const wrapper = shallow(
        <NotFoundPage/>
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
   
  test("Returned value", () => {
    let  wrapper = mount(<NotFoundPage/>);
    expect(wrapper.text()).toBe("404 Not found Page!");
  });

});