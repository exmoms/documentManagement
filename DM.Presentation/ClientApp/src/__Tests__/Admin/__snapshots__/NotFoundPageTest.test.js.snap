// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage matches snapshot 1`] = `
<div>
  404 Not found Page!
</div>
`;
